aws/data_source_aws_organizations_organizational_unit_test.go

```diff
@@ -0,0 +1,56 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccDataSourceAwsOrganizationsOrganizationalUnit_basic(t *testing.T) {
+	resourceName := "aws_organizations_organizational_unit.test_ou2"
+	dataSourceName := "data.aws_organizations_organizational_unit.test"
```

Comment:
Only noting once in this file but should be updated everywhere:

```suggestion
	dataSourceName := "data.aws_organizations_organizational_units.test"
```